events/2019/11-contributor-summit/README.md

```diff
@@ -1 +1,50 @@
 # 2019 Kubernetes Contributor Summit NA
+
+## *This event is still in the planning stages. Please check back here for updates!*
+
+## What
+
+This event brings together new and current Kubernetes contributors to connect and learn from one another. It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice. The summit spans two days - an optional social event in the evening of Sunday, November 17th along with the main full-day event on Monday, November 18th.
+
+## Registration
+
+- [Sign the CLA](/CLA.md) if you have not done so already.
+- Registration Form Coming Soon
+
+## When and Where
+
+The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon NA](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/), make sure you plan travel accordingly.
+
+- Sunday, November 17th, 2019 - Social event (optional)
```

Comment:
put a tbd time range so people know when to plan flights. something like ~5pm-8pm tba

Reply:
Done :)